MobiusCore/Source/EffectHandlers/EffectHandlerComposition.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2019 Spotify AB.
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+public extension EffectHandler {
+    /// Merge a group of `EffectHandler`s.
+    /// Note: Any given `Effect` can be handled by at most  __1__ `EffectHandler`.
```

Comment:
This could be phrased more strongly, indicating that  a runtime error will happen if there are collisions.

Reply:
Good point